modules/administration-guide/partials/proc_importing-extensions-from-the-open-vsx-registry.adoc

```diff
@@ -3,16 +3,49 @@
 [id="importing-extensions-from-the-open-vsx-registry"]
 = Importing extensions from the Open VSX Registry
 
-In the intro:
-
-* Please clarify if this procedure is about adding extensions from https://open-vsx.org/ to Che's embedded Open VSX Registry.
-
-* Please also clarify how the list of extensions that can be added from Che's embedded Open VSX Registry is synchronized with the list of extensions at https://open-vsx.org/.
+This section describes how to add VS Code extensions from link:https://open-vsx.org/[Open VSX Registry] to {prod-short}'s embedded Open VSX Registry.
 
 .Prerequisites
 
-?
+. The extension should be published on link:https://open-vsx.org/[Open VSX Registry]
+. An information about an ID of the extension. The ID contains the extension's pubisher and name (e.g. `redhat.vscode-xml`).
 
 .Procedure
 
-?
+. Modify `openvsx-sync.json` file in the link:https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json[plugin-registry repository] to add an information about the extension:
```

Comment:
```suggestion
. Modify the plug-in registry's link:https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json[`openvsx-sync.json`] file to add an extension:
```

Reply:
Note that this is a link to the downstream file... we probably need a new antora variable for this URL so we can pull either https://github.com/eclipse-che/che-plugin-registry/blob/main/openvsx-sync.json or https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json. 

Or should we link to the raw file instead?

https://raw.githubusercontent.com/eclipse-che/che-plugin-registry/main/openvsx-sync.json
vs.
https://raw.githubusercontent.com/redhat-developer/devspaces/devspaces-3-rhel-8/dependencies/che-plugin-registry/openvsx-sync.json